core/metacling/src/TClingValue.cxx

```diff
@@ -44,6 +44,25 @@ TClingValue& TClingValue::operator=(TClingValue &Other) {
    return *this;
 }
 
+std::pair<std::string, std::string> TClingValue::ToTypeAndValueString() const {
+  std::string output = ToString();
+  int pos, bracket = 0, str_size = output.size() - 1;
```

Comment:
Why not have `str_size = output.size()` and the have ` < str_size` in the for loop?